app/src/main/java/samyups/example/trainingss/data/Dogs.kt

```diff
@@ -0,0 +1,10 @@
+package samyups.example.trainingss.data
+
+// model class
+
+data class Dogs(
```

Comment:
I would call this class `Dog` instead of `Dogs` because this class defines one single instance of a dog.